avalon/tools/loader/widgets.py

```diff
@@ -1257,14 +1282,24 @@ def change_visibility(self, column_name, visible):
         lib.change_visibility(self.model, self.tree_view, column_name, visible)
 
 
-def _load_representations_by_loader(loader, repre_ids, dbcon,
+def _load_representations_by_loader(loader, repre_contexts,
                                     options=None,
                                     data_by_repre_id=None):
-    """Loops through list of repre ids and loads them with one loader"""
+    """Loops through list of repre_contexts and loads them with one loader
+
+        Args:
+            loader (cls of api.Loader) - not initialized yet
+            repre_contexts (dicts) - full info about selected representations
+                (containing repre_doc, version_doc, subset_doc, project info)
+            options (dict) - qargparse arguments to fill OptionDialog
+            data_by_repre_id (dict) - additional data applicable on top of
+                options to provide dynamic values
+    """
     error_info = []
-    repre_contexts = pipeline.get_repres_contexts(repre_ids, dbcon)
 
-    options = options or {}
+    if options is None:  # not load when cancelled
```

Comment:
In that case `options` is required argument of function and should not have default value `None`.